src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+	private List<Car> cars;
+
+	public Cars(int carCount) {
+		this.cars = new ArrayList<>();
+		for(int i = 0; i < carCount; i++) {
+			cars.add(new Car());
+		}
+	}
+
+	public void moveCars() {
```

Comment:
```suggestion
	public void move() {
```
cars 객체의 메서드이기 때문에 반복적으로 Cars를 붙일 필요는 없어 보이네요